### PR TITLE
CARMA Messenger - Truck Inspection UI - Update

### PR DESCRIPTION
**Pull request by dan-du-car**
https://usdot-carma.atlassian.net/browse/CAR-2123

### CODE REVIEW

**Comment by dan-du-car** [comment]
Workspace instruction
mkdir -p ~/carma_messenger_ws/src
cd ~/carma_messenger_ws/src
git clone https://github.com/usdot-fhwa-stol/carma-messenger
cd ~/carma_messenger_ws
catkin_make
source ~/carma_messenger_ws/devel/setup.bash
roslaunch truck_inspection_plugin truck_inspection_plugin.launch
-- start apache2 and make sure the website folder is configured as the apache2 root folder 
---running apache2
service apache2 start
---open a browser 
http://localhost

**Comment by dan-du-car** [comment]
Linked issues:
https://github.com/usdot-fhwa-stol/carma-platform/issues/687

File: carma-messenger-ui/website/scripts/control.js [hunk 1]
```diff
@@ -149,11 +149,21 @@ function createDivFramePanelText(divFrameId,divFrameCss,divPanelId,divPanelCss,s
...
-    divText.id=divTextId;
     if(divTextValue!=null && divTextValue!="undefined" && divTextValue.length > 0)
-    {
-        divText.innerText=divTextValue;
+    { 
+        if(divTextValue.includes("../../images/")){
+            var img = document.createElement('img');
+            //image path
+            img.src=divTextValue; 
+            img.alt="Image is not avaialbe";
```
[the remark this answers is not in the document]
Comment (dan-du-car) [reply carma-messenger-ui/website/scripts/control.js:160]: It is updated. Please see:
https://github.com/usdot-fhwa-stol/carma-messenger/commit/f7e6dcf3b518961f1493992815f5dc3c852908c6

File: carma-messenger-ui/website/scripts/control.js [hunk 2]
```diff
@@ -1,577 +1,577 @@
...
+    var divText = document.createElement('div');
+    divText.id=divTextId;    
+    divText.className=divTextCSS;
+    if(divTextValue!=null && divTextValue!="undefined" && divTextValue.length > 0)
+    { 
+        if(divTextValue.includes("../../images/")){
+            var img = document.createElement('img');
+            //image path
+            img.src=divTextValue; 
+            img.alt="Image is not availabe";
```
Comment (maefromm) [inline carma-messenger-ui/website/scripts/control.js:160]: still incorrect. Needs the letter L after the letter b. Should say "Image is not available."

**Comment by dan-du-car** [comment]
> sorry still incorrect spellin

It is addressed. Thanks.

**Pull request merged**